sdcm/sct_provision/aws/cluster.py

```diff
@@ -130,6 +130,16 @@ def _user_data(self) -> str:
     def _regions(self) -> List[str]:
         return self.params.region_names
 
+    @cached_property
+    def _regions_with_nodes(self) -> List[str]:
+        output = []
+        for region_id, region_name in enumerate(self.params.region_names):
+            if len(self._node_nums) <= region_id:
+                continue
```

Comment:
I guess it is going to ignore nodes in second+ region because we compare here it's index to the number of nodes.
So, second region with one going to be ignored if I understand the logic correctly.

Or I am missing something here?

Reply:
Nope, self._node_nums is a list, this line is intended to skip regions for whose we don't have node numbers

Reply:
Ok